app.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+var userName = prompt('What should I call you?');
+  alert('Okay, ' + userName + ', my name is Sean.');
+
+var userAck = prompt('Before you READ about me, let\'s play a game to get to know each other, shall we?');
+  if (userAck.toLowerCase() === 'yes' || userAck.toLowerCase() === 'y') {
+  alert('Let\'s do this!');
+} else if (userAck.toLowerCase() === 'no' || userAck.toLowerCase () === 'n') {
+  alert('Okay, fine, be that way...but I\'m still going to bother you with questions.');
+} else if (userAck !==true){
+  alert('Okaaaaay, let\'s move on');
+}
```

Comment:
consider adding a line of white space after your final `else` statement and the variable declaration of your next question to break up the code.